docs/index.rst

```diff
@@ -5,65 +5,84 @@ Tahoe-LAFS
 .. Please view a nicely formatted version of this documentation at
    http://tahoe-lafs.readthedocs.io/en/latest/
 
-Contents:
+Tahoe-LAFS is a Free and Open decentralized cloud storage system.
```

Comment:
I might leave "cloud" out here.  "decentralized" probably gives enough of the right flavor?  Or maybe this text should be exactly the text from the "about" doc (either that text copied here, this text copied there, or some sphinx trick to render the text in both places).

Reply:
Dropped "cloud" in 14fa9ed.  I copied that text from the website.

I don't know if Sphinx can include a fragment from another doc, but it has something called "substitutions".  I am not too sure about using that here.

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#substitutions

Reply:
Sphinx can include portions of another document, like this:

```
.. include:: about.rst
   :start-line: 9
   :end-line: 20
```

That will include the below text from `about.rst` in index page:

```
Welcome to Tahoe-LAFS_, the first decentralized storage system with
*provider-independent security*.

Tahoe-LAFS is a system that helps you to store files. You run a client
program on your computer, which talks to one or more storage servers on other
computers. When you tell your client to store a file, it will encrypt that
file, encode it into multiple pieces, then spread those pieces out among
multiple servers. The pieces are all encrypted and protected against
modifications. Later, when you ask your client to retrieve the file, it will
find the necessary pieces, make sure they haven't been corrupted, reassemble
them, and decrypt the result.

```

That is a little too wordy.  The text that is already here (borrowed from [README.rst](https://github.com/tahoe-lafs/tahoe-lafs/blob/master/README.rst)) feels shorter, and maybe that would be the right text to have on this page:

```
Tahoe-LAFS is a Free and Open decentralized storage system.
It distributes your data across multiple servers.
Even if some of the servers fail or are taken over by an attacker,
the entire file store continues to function correctly,
preserving your privacy and security.
```

Or maybe the actual content of this text should be outside the scope of this PR?  We could not add any new text to the index page in this PR, and just stick with reorganization of table of contents.